project.go

```diff
@@ -22,12 +22,12 @@ type projectContainer struct {
 
 func newProject(path string) (projectContainer, error) {
 	if !xcodeproject.IsXcodeProj(path) {
-		return projectContainer{}, fmt.Errorf("path (%s) is not a Project", path)
+		return projectContainer{}, fmt.Errorf("%s is not an xcode project", path)
```

Comment:
xcode -> Xcode officially I beleive

Reply:
intentionally made everything lower case. Do you think we should use `Xcode` in any case?

Reply:
I prefer the 'correct' uppercase capitalization.

Reply:
Updated